sql/core/src/main/scala/org/apache/spark/sql/RuntimeConfig.scala

```diff
@@ -60,6 +60,16 @@ class RuntimeConfig private[sql](sqlConf: SQLConf = new SQLConf) {
     set(key, value.toString)
   }
 
+  /**
+   * Sets the given Spark runtime configuration property.
+   *
+   * @since 3.4.0
+   */
+  def set[T](entry: ConfigEntry[T], value: T): Unit = {
```

Comment:
`ConfigEntry` is a private class, that's why `protected[sql] def get[T](entry: ConfigEntry[T]): T` is not public. I think we should do the same for `set`.

Reply:
Sure, I will change it.